osmoutils/accum/export_test.go

```diff
@@ -75,3 +77,15 @@ func (accum *AccumulatorObject) SetValue(value sdk.DecCoins) {
 func (o *Options) Validate() error {
 	return o.validate()
 }
+
+// WARNING: only used in tests to make sure that receiver is mutated.
+// Do not move out of export_test.go and do not use in production code.
+func (accum AccumulatorObject) GetTotalShareField() sdk.Dec {
+	return accum.totalShares
+}
+
+// WARNING: only used in tests to make sure that receiver is mutated.
+// Do not move out of export_test.go and do not use in production code.
+func (accum AccumulatorObject) GetValueField() sdk.DecCoins {
+	return accum.value
+}
```

Comment:
Why would getters ever be dangerous?

Reply:
We already have `GetValue` and `GetTotalShare` getters that read from the state instead of grabbing this value. Just left a note in case someone tries to duplicate it.